awx/main/analytics/collectors.py

```diff
@@ -334,7 +334,8 @@ def unified_jobs_table(since, full_path, until, **kwargs):
                                  main_unifiedjob.finished, 
                                  main_unifiedjob.elapsed, 
                                  main_unifiedjob.job_explanation, 
-                                 main_unifiedjob.instance_group_id
+                                 main_unifiedjob.instance_group_id,
```

Comment:
@Ladas do we need to increment the version in the `@register()` decorator above?  What's the protocol for when we want to make/additions changes to this schema?

Reply:
@ryanpetrello  Yeah, the version increment looks good.

There isn't any official protocol right now, we're keeping track of all changes and test them on analytics side. :-) (and we're trying to create automated tests, that would test all supported Tower/Ansible versions)